pages/jdl.md

```diff
@@ -1,1447 +0,0 @@
-
-    entity A {
-      name String required
-    }
-    entity B
-    entity C
-
-    dto A, B with mapstruct
-
-    paginate A with infinite-scroll
-    paginate B with pagination
-    paginate C with pager  // pager is only available in AngularJS
-
-    service A with serviceClass
-    service C with serviceImpl
-
-The keywords `dto`, `paginate`, `service` and `with` were added to the grammar to support these changes.
-If a wrong option is specified, JDL will inform you of that with a nice, red message and will just ignore it so as not
-to corrupt JHipster's JSON files.
-The complete list of available options is [here](#entity_options).
-
-#### Service option
-
-No services specified will create a resource class which will call the repository interface directly. This is the
-default and simplest option, see A.
-Service with serviceClass (see B) will make the resource call the service class which will call the repository interface.
-Service with serviceImpl (see C) will make a service interface which will be used by the resource class.
-The interface is implemented by an concrete class which will call the repository interface.
-
-Using no service unless sure is the simplest option and good for CRUD. Use service with a Class if you will have a lot
-of business logic which will use multiple repositories making it ideal for a service class. Jhipsters are not fan of
-unnecessary Interfaces but if you like them go for service with impl.
-
-    entity A
-    entity B
-    entity C
-
-    // no service for A
-    service B with serviceClass
-    service C with serviceImpl
-
-
-JDL also supports mass-option setting. it is possible to do:
-
-    entity A
-    entity B
-    ...
-    entity Z
-
-    dto * with mapstruct
-    service all with serviceImpl
-    paginate C with pagination
-
-Note that `*` and `all` are equivalent.
-Latest version introduces exclusions (which is quite a powerful option when setting options for every entity):
-
-    entity A
-    entity B
-    ...
-    entity Z
-
-    dto * with mapstruct except A
-    service all with serviceImpl except A, B, C
-    paginate C with pagination
-
-
-With JHipster, you can also tell whether you don't want any client code, or server code.
-Even if you want to add a suffix to Angular-related files, you can do that in JHipster.
-[
-Filtering](https://www.jhipster.tech/entities-filtering/) options can be activated on a per entity basis: filter `<entity name>` or for all entities: filter `*`.
-In your JDL file, simply add these lines to do the same:
-
-```
-entity A
-entity B
-entity C
-
-skipClient A
-skipServer B
-angularSuffix * with mySuperEntities
-filter C
-```
-
-Finally, table names can also be specified (the entity's name will be used by default):
-
-```
-entity A // A is the table's name here
-entity B (the_best_entity) // the_best_entity is the table's name
-```
-
-### <a name="annotations"></a> Annotations
```

Comment:
Seems like we don't have any doc about the custom annotation support

Reply:
True, I'll add something in the option information page.